File: prak204/PRAK204-2010817320004-SalsabilaSyifa.py
import math

jari = float(input("Masukan Jari-jari Bejana : "))
tinggi = float(input("Masukan Tinggi Bejana : "))

luas = 2 * math.pi * jari * (jari + tinggi)
volume = math.pi * jari * jari * tinggi
keliling = 2 * ((2*math.pi*jari) + tinggi)

print("Luas Tabung = ",format(luas,'.2f'))
print("Volume Tabung = ",format(volume,'.2f'))
print("Keliling Tabung = ",format(keliling,'.2f'))

File: prak204/PRAK204-2010817320004-SalsabilaSyifa.c
#include <stdio.h>
#include <math.h>
#define _USE_MATH_DEFINES

int main () {
    M_PI;
    float jari, tinggi, volume, keliling, luas;
    
    printf("Masukan Jari-Jari Bejana : ");
    scanf("%f", &jari);

    printf("Masukan Tinggi Bejana : ");
    scanf("%f", &tinggi);

    luas = 2 * M_PI * jari * (jari + tinggi);
    printf("Luas Bejana : %.2f\n", luas);

    volume = M_PI * jari * jari * tinggi ;
    printf("Volume Bejana : %.2f\n", volume);

    keliling = 2 * ((M_PI * jari) + tinggi);
    printf("Keliling Bejana : %.2f\n", keliling);

    return 0;
}